README.md

```diff
@@ -1,231 +1,204 @@
 # fastify-autoload
```

Comment:
I'm not seeing any esm examples in the readme (sorry if I missed them) - can we please add esm examples (especially for autoPrefix and autoConfig)

Reply:
actually I wonder if it might be word having a `README.esm.md` which is an esm version of this readme and then link to it from here

eventually we'd make README.esm.md the README.md and move README.md to README.cjs.md